components/clm/src/biogeochem/CNPhenologyBeTRMod.F90

```diff
@@ -1347,6 +1356,14 @@ subroutine CropPhenology(num_pcropp, filter_pcropp                     , &
     real(r8) dayspyr  ! days per year
     real(r8) crmcorn  ! comparitive relative maturity for corn
     real(r8) ndays_on ! number of days to fertilize
+    logical p_season  ! precipitation seasonal
+    logical t_season  ! temperature seasonal
+    logical no_season ! neither temperature or precipitation seasonal
+    real(r8), parameter :: tcvp = 0.4       ! threshold of CV of precipitation to indicate seasonality
```

Comment:
Seems like you are defining `tcvp` and `tcvt` also in https://github.com/E3SM-Project/E3SM/pull/2999/files#diff-ac10333178760bf1fc41e8763b73a080R114. Is it possible to consolidate all those definition in a single module and include them via `use <module>`?

Reply:
OK, I pulled tcvp, tcvt, and cst (the temperature variable 283._r8) into the CropType and defined them as a parameter there.